### DISCUSSION TITLE
Model with multiple features

### DISCUSSION THREAD

**Discussion by Aishuvenkat09** [Project-MONAI/MONAI#3981] [Q&A]
Hi ! Maybe a rookie question but how do I customize the model to take two input features 

I'm working on an BoneAge Prediction problem. I want to feed the model with 2 inputs (Image and Gender) but not sure how to do that. Do I have to completely customize the monai model architecture to let the forward() function to take multiple inputs ? Any resources to such usecase would be a great help thank you!

**Comment by Nic-Ma** [comment]
Hi @Aishuvenkat09 ,

Thanks for your interest here.
May I know what network you are trying to use for your task?

Thanks.

**Comment by Aishuvenkat09** [comment]
I’m just trying out with DenseNet but ideally I would like use Pretrained Vgg16, inception and xception models to benchmark performance compared to models built on tensorflow.

Regards,
Aishwarya Vantipuli
________________________________

**Comment by ebrahimebrahim** [comment]
What you are trying to do is called multimodal fusion, so that would be a good thing to read up on.

This article is useful and lists some of your options: https://www.nature.com/articles/s41746-020-00341-z

**Comment by Aishuvenkat09** [comment]
Thank you @ebrahimebrahim. I have done something similar to early fusion in PyTorch. But I want to do the same in MONAI but not sure how to do that.
Also, can I use any network like (inception) and implement that in MONAI ? or is it advisable to use MONAI networks to get most out of them.
